app/models/export/anything.rb

```diff
@@ -0,0 +1,67 @@
+module Export
+  class Anything
+    attr_reader :output_name
+
+    def initialize(relation, output_name = :tmp)
+      @relation = relation
+      @output_name = (output_name || :tmp).to_sym
+    end
+
+    def run
+      if @relation.empty?
+        STDERR.puts 'No records to export'
```

Comment:
given that it's likely this code will be run with using the "all" rake task perhaps it would be useful for the user to know exactly which model there are no records for?

Reply:
Will do.